src/modules/layout.js

```diff
@@ -0,0 +1,55 @@
+import Storage from './localStorage.js';
+
+class Lists {
+  static displayToDo() {
+    const todo = Storage.getLists();
+
+    todo.forEach((todo) => {
+      Lists.addLList(todo);
+    });
+  }
+
+  static addLList(todo) {
+    const ulContainer = document.getElementById('tbody');
+    const edite = document.createElement('tr');
+
+    /* edite.innerHTML = `

```

Comment:
Kindly remove this block of code too

Reply:
Thanks a lot  for your time @limavfabio, I will begin work on these issues as soon as possible.